src/main/kotlin/io/github/moulberry/notenoughupdates/miscfeatures/CountdownCalculator.kt

```diff
@@ -73,6 +80,7 @@ class CountdownCalculator {
             else -> return
         }
         if (event.itemStack != null && Minecraft.getMinecraft().thePlayer?.openContainer != null) {
+            if (event.itemStack.displayName.endsWith("'s Party")) return //TODO: why does this event falsely activate in party finder? -ery
```

Comment:
it activates because ` ($e` (see `CALENDARDETAILS`) matches for the level of the class 
e.g. `myname12s: Mage (24)`: the ` (24)` matches.

Instead of fixing this only for the party finder you could add a check for the displayname in the calendar (it always starts with `Day x`)

Reply:
`if (countdownKind == CountdownTypes.CALENDARDETAILS && !event.itemStack.displayName.startsWith("§aDay ")) return`